Tests fails with macos on arm64

The following tests are failing on macos with arm64. It feels like the cpu architecture is not discovered properly.

* ClientIT.testContainer:56
* ClientIT.testDirectory:23
* ClientIT.testError:78
* ClientIT.testGit:35
* ClientIT.testList:96

```
[ERROR] org.chelonix.dagger.client.ClientIT.testGit  Time elapsed: 0.614 s  <<< ERROR!
java.io.IOException: Could not find checksum for dagger_v0.6.2_darwin_unknown.tar.gz
	at org.chelonix.dagger.client.engineconn.CLIDownloader.downloadCLI(CLIDownloader.java:71)
	at org.chelonix.dagger.client.engineconn.CLIDownloader.downloadCLI(CLIDownloader.java:54)
	at org.chelonix.dagger.client.engineconn.CLIDownloader.downloadCLI(CLIDownloader.java:61)
	at org.chelonix.dagger.client.engineconn.Connection.getCLIPath(Connection.java:124)
	at org.chelonix.dagger.client.engineconn.Connection.fromCLI(Connection.java:147)
	at org.chelonix.dagger.client.engineconn.Connection.get(Connection.java:161)
	at org.chelonix.dagger.client.Dagger.connect(Dagger.java:25)
	at org.chelonix.dagger.client.Dagger.connect(Dagger.java:15)
	at org.chelonix.dagger.client.ClientIT.testGit(ClientIT.java:35)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.junit.platform.commons.util.ReflectionUtils.invokeMethod(ReflectionUtils.java:727)
...
[INFO] 
[INFO] Results:
[INFO] 
[ERROR] Errors: 
[ERROR]   ClientIT.testContainer:56 » IO Could not find checksum for dagger_v0.6.2_darwin_unknown.tar.gz
[ERROR]   ClientIT.testDirectory:23 » IO Could not find checksum for dagger_v0.6.2_darwin_unknown.tar.gz
[ERROR]   ClientIT.testError:78 » IO Could not find checksum for dagger_v0.6.2_darwin_unknown.tar.gz
[ERROR]   ClientIT.testGit:35 » IO Could not find checksum for dagger_v0.6.2_darwin_unknown.tar.gz
[ERROR]   ClientIT.testList:96 » IO Could not find checksum for dagger_v0.6.2_darwin_unknown.tar.gz
[INFO] 
[ERROR] Tests run: 5, Failures: 0, Errors: 5, Skipped: 0
```

--- a/dagger-java-sdk/src/main/java/org/chelonix/dagger/client/engineconn/CLIDownloader.java
+++ b/dagger-java-sdk/src/main/java/org/chelonix/dagger/client/engineconn/CLIDownloader.java
@@ -192,6 +192,8 @@ class CLIDownloader {
             return "x86";
         } else if (arch.contains("arm")) {
             return "armv7";
+        } else if (arch.contains("aarch64")) {
+        	return "arm64";
         } else {
             return "unknown";
         }

